test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
```

Comment:
`test/idl/validate.js` call `run()` and when I wrote that I worried about what a second test requiring async setup should do. This seems to work, but I don't understand why. Adding an artificial delay to `css.parseAll()`, the tests still don't run until this has run. I don't understand how that works.

Since it works I guess that's good, but if someone does understand how this works, I'd like to know :)

Reply:
I had noticed the `run()` in your tests and actually did not understand what that `run()` function was referring to, so I dropped it and that did not seem to have any consequence. Thinking about it some more, I don't really understand why this works either. I'll look into it.

Reply:
This is using https://mochajs.org/#delayed-root-suite, but that documentation alone really provides no clues about why it doesn't break when the promises resolve in the wrong order...

Reply:
And not calling `run()` does have consequences: the tests won't run if they are run in isolation, e.g. with `mocha --recursive --delay test/css/`. Which makes complete sense. I'll add the missing `.then(run)`.